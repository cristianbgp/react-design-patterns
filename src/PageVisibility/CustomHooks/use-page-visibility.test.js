import React from "react";
import { render } from "@testing-library/react";

import usePageVisibility from "./use-page-visibility";

test("Page visibility with custom hooks", () => {
  function Component() {
    const status = usePageVisibility();
    return (
      <>
        <h1>Using Custom Hooks</h1>
        <p>
          Page Visibility: <span>{status.toString()}</span>
        </p>
      </>
    );
  }
  const { asFragment } = render(<Component />);

  expect(asFragment()).toMatchSnapshot();
});
