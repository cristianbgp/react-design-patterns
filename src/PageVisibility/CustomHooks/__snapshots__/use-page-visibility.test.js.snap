// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page visibility with custom hooks 1`] = `
<DocumentFragment>
  <h1>
    Using Custom Hooks
  </h1>
  <p>
    Page Visibility: 
    <span>
      true
    </span>
  </p>
</DocumentFragment>
`;
